### PR TITLE
align Reactive Management definition with ATIS's ...

### PR DESCRIPTION
**Pull request by danwing**
document ATIS-I-0000096, 'Content Classification for Traffic Optimization', available at https://atis.org/whitepapers/white-papers-a-z

### CODE REVIEW

**Review by sridharangirish91 — APPROVED** [review]

File: draft-kwbdgrr-tsvwg-net-collab-rqmts.md [hunk 1]
```diff
@@ -299,8 +299,7 @@ Intentional Management:
 quota or bandwidth limit, or quality (delay and/or jitter)) assurances.
 
 Reactive Management:
-: Network reactions to congestion events or
-protection polices under attacks, with very short to very long
+: network policies which react to congestion events with very short to very long
```
Comment (boucadair) [inline draft-kwbdgrr-tsvwg-net-collab-rqmts.md]:
```suggestion
: Network management actions that are undertaken as a reaction to unplanned overload events. Concretely, this includes policies which react to congestion events with very short to very long durations (e.g., varying wireless and mobile air interface conditions) or protection policies to soften the impact of ongoing attacks.
```

File: draft-kwbdgrr-tsvwg-net-collab-rqmts.md [hunk 2]
```diff
@@ -299,8 +299,7 @@ Intentional Management:
...
 durations (e.g., varying wireless and mobile air interface conditions).
```
Comment (boucadair) [inline draft-kwbdgrr-tsvwg-net-collab-rqmts.md]:
```suggestion
```

File: draft-kwbdgrr-tsvwg-net-collab-rqmts.md (hunk 1 above)
Comment (boucadair) [reply draft-kwbdgrr-tsvwg-net-collab-rqmts.md]: Dan's def is restrictive, IMO hence the proposed revised text.

**Review by boucadair — CHANGES_REQUESTED** [review]

File: draft-kwbdgrr-tsvwg-net-collab-rqmts.md (hunk 1 above)
Comment (John-Kaippallimalil) [reply draft-kwbdgrr-tsvwg-net-collab-rqmts.md]: Med's revision is clear.

**Review by boucadair — APPROVED** [review]

**Pull request merged**